TACC/tup-ui
# Task/tup 375 header for project detail

## Overview:
Create project header when an active project is selected. Created tests and created Route to projects/active in App.jsx

## Related:

- [TUP-375](https://jira.tacc.utexas.edu/browse/TUP-375)

## Changes:

## Testing:

1. Go to [http://localhost:8000/dashboard/projects](http://localhost:8000/dashboard/projects)
2. Click on a project title
3. Check that the right fields display. Compare to your [TACC User Portal](https://portal.tacc.utexas.edu/projects-and-allocations) to make sure the values are correct.
4. Unix Group and Field of Science will not display yet.
5. Click Active Projects link and make sure it takes you back to a listing of all projects

## UI:

<img width="1368" alt="Screen Shot 2023-01-20 at 12 28 21 PM" src="https://user-images.githubusercontent.com/96220951/213778076-a3f83058-a377-43a6-9e3e-28d8ca95ea1f.png">


## Notes:

TODO:

- [x] Need to get a new endpoint that retrieves field of science from TAS

- [x] Find out why Unix Group is adding } at the end of the number when returned from object. Tried slicing it off but it slices off the last number rather than the bracket.
